application/comit_node/src/swap_protocols/rfc003/state_machine.rs

```diff
@@ -1,5 +1,4 @@
-// TODO: Figure out how to properly place this on the state_machine_future
-// derive so that is is forwarded to the generated structs and impl
+// The state_machine_future derive generates quite complex code ...
```

Comment:
:upside_down_face: :upside_down_face: :upside_down_face: 
```suggestion
// The state_machine_future derive generates quite complex code...
```